package-lock.json

```diff
@@ -3269,7 +3269,8 @@
         "console-control-strings": {
           "version": "1.1.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
```

Comment:
Are you using the latest NPM here? I don't get the same changes when I install it.

Reply:
I've updated using the latest npm. I can't keep up!